packages/browser/src/Component/ItownsUtil.js

```diff
@@ -1,68 +1,29 @@
+// If functions declare here are needed by widgets these functions should be propose to itowns
+// If not this function are just meant to be use in an @ud-viz/browser eg in Template
+
 const THREE = require('three');
 const itowns = require('itowns');
-import { TilesManager, LayerManager } from './Itowns';
 
-/**
- * These extensions should belong elsewhere since it should be possible
- * to manipulate Temporal 3DTiles without having a dependence to its widget...
- */
-import { $3DTemporalBatchTable } from '../Widget/Temporal/Model/3DTemporalBatchTable';
-import { $3DTemporalBoundingVolume } from '../Widget/Temporal/Model/3DTemporalBoundingVolume';
-import { $3DTemporalTileset } from '../Widget/Temporal/Model/3DTemporalTileset';
+/**  ADD LAYER TO ITOWNS VIEW FROM CONFIG */
 
 /**
  * It creates a 3D Tiles layer,
  * and adds it to the layer manager
  *
  * @param {*} layer - the layer object from the config file
- * @param  {LayerManager} layerManager - the layer manager object
  * @param {itowns.View} itownsView - the itowns view
+ * @param {object} extensions - optional extensions
  * @returns {itowns.C3DTilesLayer} A 3D Tiles Layer
```

Comment:
the return seems to be deprecated

Reply:
It's actually up to date, createC3DTilesLayer, previous setupC3DTilesLayer now return a layer